package.json

```diff
@@ -2,11 +2,13 @@
 	"name": "faucet-pipeline-static",
 	"version": "2.1.0",
 	"description": "static files for faucet-pipeline",
-	"main": "index.js",
+	"main": "lib/index.js",
```

Comment:
Minor detail, just because these days I try to be a little bit more explicit:

```suggestion
	"main": "./lib/index.js",
```

Reply:
:heavy_check_mark: